internal/cli/command/controlplane/exec.go

```diff
@@ -188,6 +198,53 @@ func pullImage(options *cpContext, _ cli.Cli) error {
 	return cmd.Run()
 }
 
+func fetchContainerCommandOutputAndError(options *cpContext, cmd []string, cmdEnv map[string]string) string {
+	out, err := fetchContainerCommandOutput(options, cmd, cmdEnv)
+	if err != nil {
+		out += "\n\n" + err.Error()
+	}
+	return out
+}
+
+func fetchContainerCommandOutput(options *cpContext, cmd []string, cmdEnv map[string]string) (string, error) {
```

Comment:
**Note**: I would prefer emphasizing container command execution here, such as `runContainerCommand(...)`, though I understand why this name is also valid.